python_scripts/trees.py

```diff
@@ -326,13 +327,17 @@ def plot_decision_function(X, y, clf, ax=None):
 # in the partition >= 42 mm, we cannot differentiate well between Gentoo and
 # Chinstrap (while they are almost twice more Gentoo).
 #
-# We should use a statistical measure that uses the all class
+# We should use a statistical measure that uses all the class
 # probabilities, as the criterion to qualify the purity
-# of a partition. We will choose as an example the entropy criterion (also used
+# of a partition (which will be a leaf in a decision tree).
```

Comment:
Partitions could be either leaf or split nodes? It's just the result of a split?

Reply:
Yes right ! Partition here is a part of the full data.

Reply:
Right, Partitions could be either leaf or split nodes